### PR TITLE
Removed dollar signs

### LINKED ISSUE #120: Dollar Sign in Code Blocks Gets Copied with Copy Button
When using the copy function on the website for CLI commands, the dollar sign "$" is also copied over as well

### PR DESCRIPTION
**Pull request by CharlieMoon37**
This PR removes the dollar signs and venv prompts as these get copied over

Signed-off-by: Charlie Moon <charlie@chaosiq.io>

closes #120

### CODE REVIEW

File: sources/reference/tutorial.md [hunk 1]
```diff
@@ -195,13 +195,13 @@ You may now run the application.
...
 ```console
-$ cp valid-cert.pem cert.pem
+cp valid-cert.pem cert.pem
 ```
 
 Next, start the services, in one terminal:
 
 ```console
-$ python3 astre.py
+python3 astre.py
```
Comment (ciaranevans) [inline sources/reference/tutorial.md:204]: Ditto command & output

File: sources/reference/tutorial.md [hunk 2]
```diff
@@ -212,7 +212,7 @@ $ python3 astre.py
 Then, in another terminal:
 
 ```console
-$ python3 sunset.py
+python3 sunset.py
```
Comment (ciaranevans) [inline sources/reference/tutorial.md:215]: Ditto command & output

File: sources/reference/tutorial.md [hunk 3]
```diff
@@ -224,7 +224,7 @@ $ python3 sunset.py
 Now you may perform a simple call:
 
 ```console
-$ curl -k https://localhost:8443/city/Paris
+curl -k https://localhost:8443/city/Paris
```
Comment (ciaranevans) [inline sources/reference/tutorial.md:227]: Ditto command & output

File: sources/reference/tutorial.md [hunk 4]
```diff
@@ -237,7 +237,7 @@ chained together over HTTPS.
 Now your application is running, execute the experiment once again:
 
 ```console
-$ chaos run experiment.json
+chaos run experiment.json
```
Comment (ciaranevans) [inline sources/reference/tutorial.md:240]: Ditto command & output

File: sources/reference/tutorials/extend.md [hunk 5]
```diff
@@ -27,7 +27,7 @@ fn main() {
 ```
 
 ```console
-$ rustc echo.rs
+rustc echo.rs
```
Comment (ciaranevans) [inline sources/reference/tutorials/extend.md:30]: Can we split command and output?

File: sources/reference/tutorials/extend.md [hunk 6]
```diff
@@ -109,7 +109,7 @@ If `kettle.py` lives next to `experiment.json` then the action could be:
 You could finally make it available as follows:
 
 ```console
-$ ls
+ls
```
Comment (ciaranevans) [inline sources/reference/tutorials/extend.md:112]: Ditto on command & output

File: sources/reference/tutorials/tolerance.md [hunk 7]
```diff
@@ -82,7 +82,7 @@ expected `"hello"`.
 Here is an example of it running:
 
 ```
-$  chaos run experiment.json
+ chaos run experiment.json
```
Comment (ciaranevans) [inline sources/reference/tutorials/tolerance.md:85]: Command and output split please

File: sources/reference/tutorial.md [hunk 8]
```diff
@@ -163,7 +163,7 @@ code of `200`. Richer tolerances can be created by using regex or jsonpath.
 To run the experiment, simply execute the following command:
 
 ```console
-$ chaos run experiment.json
+chaos run experiment.json
```
[the remark this answers is not in the document]
Comment (CharlieMoon37) [reply sources/reference/tutorial.md:166]: done

File: sources/reference/tutorial.md (hunk 1 above)
Comment (CharlieMoon37) [reply sources/reference/tutorial.md:207]: done

File: sources/reference/tutorial.md (hunk 2 above)
Comment (CharlieMoon37) [reply sources/reference/tutorial.md:221]: done

File: sources/reference/tutorial.md (hunk 3 above)
Comment (CharlieMoon37) [reply sources/reference/tutorial.md:236]: done

File: sources/reference/tutorial.md (hunk 4 above)
Comment (CharlieMoon37) [reply sources/reference/tutorial.md:249]: done

File: sources/reference/tutorials/extend.md (hunk 5 above)
Comment (CharlieMoon37) [reply sources/reference/tutorials/extend.md:30]: done

File: sources/reference/tutorials/extend.md (hunk 6 above)
Comment (CharlieMoon37) [reply sources/reference/tutorials/extend.md:124]: done

File: sources/reference/tutorials/tolerance.md (hunk 7 above)
Comment (CharlieMoon37) [reply sources/reference/tutorials/tolerance.md]: done

**Review by ciaranevans — APPROVED** [review]

**Comment by Lawouach** [comment]
I'll be honest, I prefer having the $ sign. It is clear it's a command line.

One Reliably, we managed to do a bit of css magic so that the sign isn't copied. I wish we had support for this here.

**Comment by Lawouach** [comment]
Discussed and we'll go with removing the $ sign for now

**Pull request merged**